src/host/usbh.c

```diff
@@ -889,9 +889,20 @@ TU_ATTR_FAST_FUNC void hcd_event_handler(hcd_event_t const* event, bool in_isr)
 {
   switch (event->event_id)
   {
+#if CFG_TUH_HUB
 //    case HCD_EVENT_DEVICE_REMOVE:
 //      // mark device as removing to prevent further xfer before the event is processed in usbh task
 //      break;
+#else
```

Comment:
this is a hack, we cannnot clear the event queue like this though. E.g rusb2 with 2 ports can be configured as BOTH host, but they still share the same event queue. Clearing the queue is not an option even there is no hub support. This is mostly caused by physical connection bouncing, We will analyze the rootcause and fix it properly though, just need a bit of time.